src/main/java/org/openrewrite/java/migrate/util/MigrateCollectionsSingletonList.java

```diff
@@ -54,22 +54,14 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
                 if (SINGLETON_LIST.matches(method)) {
                     maybeRemoveImport("java.util.Collections");
                     maybeAddImport("java.util.List");
-                    return getListOfTemplate().withArguments(m.getArguments()).withPrefix(m.getPrefix());
+                    return JavaTemplate.builder("List.of(\"X\")")
+                            .staticImports('java.util.List')
+                            .build()
+                            .apply(getCursor(), m.getCoordinates().replace());
```

Comment:
We'll want to add the arguments into the list here too, with something lijke
```suggestion
                    String argsPlaceHolders = String.join(",", Collections.nCopies(recipes.size(), "#{any()}"));
                    return JavaTemplate.builder("List.of(" + argsPlaceHolders + ')')
                            .staticImports('java.util.List')
                            .build()
                            .apply(getCursor(), m.getCoordinates().replace(), m.getArguments());
```